### PR TITLE
Remove Config from main class

### PR DESCRIPTION
**Pull request by nicolastarzia**
This PR would be a suggestion to change all of config params to a class, and read from then.

Indeed if we need to add more param's would be hard to find all of names that we need to added in Azure Functions.

### CODE REVIEW

File: JekyllBlogCommentsAzure/WebConfigurator.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+        public string PullRequestRepository => ConfigurationManager.AppSettings["PullRequestRepository"];
+
+        public string CommentFallbackCommitEmail => ConfigurationManager.AppSettings["CommentFallbackCommitEmail"];
+
+        public string SentimentAnalysisSubscriptionKey => ConfigurationManager.AppSettings["SentimentAnalysis.SubscriptionKey"];
+
+        public string SentimentAnalysisRegion => ConfigurationManager.AppSettings["SentimentAnalysis.Region"];
+
+        public string SentimentAnalysisLang => ConfigurationManager.AppSettings["SentimentAlaysis.Lang"];
```
Comment (Haacked) [inline JekyllBlogCommentsAzure/WebConfigurator.cs]: Heh heh, a typo here.

```suggestion
        public string SentimentAnalysisLang => ConfigurationManager.AppSettings["SentimentAnalysis.Lang"];
```

Comment (nicolastarzia) [reply JekyllBlogCommentsAzure/WebConfigurator.cs]: Ohhhhhh I didn't saw this...
Thanks to review this @Haacked

**Comment by nicolastarzia** [comment]
Awesome, thanks!!! 🤣

**Comment by damieng** [comment]
Target then manually mapping each setting can we adopt this config class approach.

https://github.com/Azure-Functions/stripe-subscription/blob/master/StripeSubscription/StripeSubscriptionFunction.cs

If like to try and keep these functions consistent if we can although that one is azure function v2 so may not be entirely possible.

**Comment by nicolastarzia** [comment]
I think would be awesome to do that.

Where did you found that not be possible in azure function v2?

**Comment by damieng** [comment]
The ConfigurationManager used in the one I linked to is v2 only :(  It also only works if the configuration variables match the property names which are not the case here.

If you can rebase what you have and drop the interface we can merge this in the way you have done it instead :)

**Comment by damieng** [comment]
Any interest in doing this?

**Comment by nicolastarzia** [comment]
Hey @damieng,
Sorry about my late response.

In the last commit I dropped the interface and rebase, could you review?

**Comment by damieng** [comment]
Looks great, thanks!

**Pull request merged**